salt/cloud/clouds/azurearm.py

```diff
@@ -163,7 +164,28 @@ def _get_active_provider_name():
         return __active_provider_name__
 
 
-@salt.utils.azurearm.deprecation_message
+def deprecation_message(function):
```

Comment:
```suggestion
def _deprecation_message(function):
```

We don't want the salt loader to include this function too :)